index.html

```diff
@@ -14,17 +14,16 @@
     <header class="page-header">
       <nav class="nav-header"> 
           <ul>
-              <li><a href="#"> Welcome </a></li>
+              <li><a href="#" class="welcome"> Welcome </a></li>
               <li> <a href="#"> <img src="/images/Union.svg" alt=""/></a></li>
           </ul>
       </nav>
     </header>
-    
-
       <section class="page-headline">
-        <h1 class="intro"> Hey there. I’m Joseph</h1>
-        <h2 class="job"> I’m a software developer</h2>
-        
+        <h1 class="intro"> Hey there. I’m Joseph 
+            <br>
```

Comment:
Kindly use margins or paddings for making spaces. `<br/>` will make issues while inspecting the styles on the browser. 😄